### PR TITLE
Created page skeleton

### PR DESCRIPTION
**Pull request by pavlentiysv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,302 @@
...
+    <title>Document</title>
+
+    <link rel="stylesheet" href="/assets/style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="content">
+        <nav>
+          <menu>
+            <li><a href="#">HOME</a></li>
```
Comment (kobweb) [inline index.html]: It's better to use `text-transform: uppercase` to make text uppercase

File: index.html [hunk 2]
```diff
@@ -1,10 +1,302 @@
...
+            <li><a href="#">PORTFOLIO</a></li>
+            <li><a href="#">BLOG</a></li>
+          </menu>
+        </nav>
+        <div class="logo">
+          <img src="img/logo.png" alt="logo" height="50px" width="50px" />
```
Comment (kobweb) [inline index.html]: Say "NO" to inline-styles. It's related to the whole document.

File: index.html [hunk 3]
```diff
@@ -1,10 +1,302 @@
...
+          <p>Expire</p>
+        </div>
+        <nav>
+          <menu>
```
Comment (kobweb) [inline index.html]: `menu` wtf?)
https://www.w3schools.com/tags/tag_menu.asp

Comment (pavlentiysv) [reply index.html]: Liza shared this link with me. So I Thought it's ok.
https://github.com/pavlentiysv/hiqo-sviridov-markup/pull/1

**Pull request merged**